### DISCUSSION TITLE
BitBar/xBar and servers/relays

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#1813] [Q&A] [answered]
Hi,
Quick question on the use of BitBar/xBar. When installed with the relevant script, xBar proposes, among others, dnscrypt and dnscrypt+quad9. Am I right in assuming that the second option would restrict the available servers to only quad9 servers while respecting filters in place? Is this compatible with anonymized dnscrypt when the list of servers (in the route) is set to '*'?
Thanks,
˜Paul

**Comment by jedisct1** [comment, marked as answer]
The second option uses whatever you have configured in `dnscrypt-proxy` (it just changes the DNS settings to `127.0.0.1`), with `9.9.9.9` as a fallback in case `dnscrypt-proxy` is not running.

**Reply by ghost** [reply]
Thanks!
